app/views/ui/register.html.haml

```diff
@@ -1,37 +1,13 @@
 %section.register.container
   .row
     .col-sm-10.col-sm-offset-1
-      %form.form-horizontal
+      = bootstrap_form_for @user, html: {class: "form-horizontal"} do |f|
```

Comment:
Rather than, `, html: {class: "form-horizontal"}` you could pass `layout: :horizontal` to the `bootstrap_form` options.